lib/Dist/Zilla/Plugin/Alien.pm

```diff
@@ -142,6 +142,12 @@ a version using the equal C<=> sign.
  bin_requires = Alien::patch
  bin_requires = Alien::gmake = 0.03
 
+=head2 stage_install
+
+If set to true Alien packages that should be installed directory into the blib
```

Comment:
Should this be:
```diff
-If set to true Alien packages that should be installed directory into the blib
+If set to true, Alien packages are installed directly into the blib
```
?

If so, the changes need to be made at <https://github.com/Perl5-Alien/Alien-Base/commit/07fe7e905abe60b44c1a3f6e56010a8921fae8b3> as well.